tests/test_functional.py

```diff
@@ -95,15 +104,17 @@ def get_tests():
 def test_functional(test_file):
     LintTest = (
         LintModuleOutputUpdate(test_file)
-        if UPDATE
+        if UPDATE.exists()
```

Comment:
@hippo91 the update file is used here in order to instantiate the right class for functional test.